src/aks-preview/azext_aks_preview/tests/latest/test_managed_cluster_decorator.py

```diff
@@ -439,6 +439,20 @@ def test_get_load_balancer_managed_outbound_ipv6_count(self):
         mc_3 = self.models.ManagedCluster(location="test_location", network_profile=network_profile_3)
         ctx_3.attach_mc(mc_3)
         self.assertEqual(ctx_3.get_load_balancer_managed_outbound_ipv6_count(), 20)
+    
+    def test_get_load_balancer_backend_pool_type(self):
+        ctx = AKSPreviewManagedClusterContext(
```

Comment:
`ctx_1`? [log](https://dev.azure.com/azure-sdk/public/_build/results?buildId=1854271&view=logs&j=59e8686e-5e74-514d-6cad-f7d66c66b425&t=59602323-6717-504d-f2cc-5c81369f935f&l=2412)

Reply:
Thanks for the catch, fixed.